pubsublite-spark-sql-streaming/pom.xml

```diff
@@ -203,6 +168,49 @@
           </execution>
         </executions>
       </plugin>
```

Comment:
nit. you can probably get rid of this, just click on the "m" in intellij and run `mvn fmt:format`